lib/std/sha1.nim

```diff
@@ -275,3 +275,32 @@ proc `==`*(a, b: SecureHash): bool =
 
   # Not a constant-time comparison, but that's acceptable in this context
   Sha1Digest(a) == Sha1Digest(b)
+
+proc isValidSha1Hash*(s: string): bool =
```

Comment:
Move this to compiler/nimblecmd.nim and the implementation can be simplified to

```nim

  result = s.len == 40 and allCharsInSet(s, HexDigits)

```

Reply:
@Araq I can move it also in the Nimble itself if you prefer?

Reply:
@Araq But I'm using it not only in `nimblecmd.nim`, but also from Nimble in `sha1hashes.nim`. It will be unnatural to import `nimblecmd.nim` there.

Reply:
I fixed the implementation with the shorter version.

Reply:
Ok, good enough.